docs/examples/rules.kts

```diff
@@ -93,7 +93,7 @@ fun PackageRule.LicenseRule.isCopyleftLimited() =
  */
 
 // Define the set of policy rules.
-val ruleSet = ruleSet(ortResult) {
+val ruleSet = ruleSet(ortResult, packageConfigurationProvider) {
```

Comment:
Commit message: Either omit "Pass", or say e.g. "Passing 'packageConfigurationProvider' to RuleSet is required since #2362."

Reply:
Done, chose the later 'Passing ...'